cardano-cli/src/Cardano/CLI/Json/Friendly.hs

```diff
@@ -2,9 +2,13 @@
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE LambdaCase #-}
 {-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE PatternSynonyms #-}
 {-# LANGUAGE ScopedTypeVariables #-}
 {-# LANGUAGE TypeApplications #-}
 {-# LANGUAGE TypeOperators #-}
+{-# OPTIONS_GHC -Wno-unrecognised-pragmas #-}
```

Comment:
Why is this required?

Reply:
Not sure to be honest. HLS inserts this when you're using `{-# HLINT ignore ...` pragma. Otherwise I'm getting a warning.

Reply:
Probably because the pragma is above `module ...` ?